vipps-order-management-api-quick-start.md

```diff
@@ -65,21 +64,24 @@ Save the following files to your computer:
 
 ## Make API calls
 
-For all of the following, you will start by sending request [Get Access Token](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Authorization-Service/operation/fetchAuthorizationTokenUsingPost).
+For all of the following, you will start by sending request `Get Access Token` which uses
+[Get Access Token][access-token-endpoint].
 This provides you with access to the API.
 
 The access token is valid for 1 hour in the test environment
 and 24 hours in the production environment.
 See the
-[API reference](https://vippsas.github.io/vipps-developer-docs/api/order-management)
+[API reference][order-mgmt-api-reference-url]
 for details about the calls.
 
-### Add receipt with order info and image
+### Add receipt with order info and image to an existing order
```

Comment:
the order doesnt need to exist yet. It is possible to add the receipt first, then create the order afterwards. But, for simplicity i guess its fine :)